docs/source/api/references/graphstorm.config.rst

```diff
@@ -7,14 +7,18 @@ GraphStorm model training and inference CLIs have a set of :ref:`built-in config
 These configurations are defined and managed in ``graphstorm.config`` module whose ``get_argument_parser()``
 method can help users to load these built-in configurations either from a yaml file or from CLI arguments.
 This ``get_argument_parser()`` is useful when users want to convert customized models to use GraphStorm CLIs.
+GraphStorm creates a ``GSConfig`` object to store the configurations loaded from a yaml file or CLI arguments,
+and then pass this ``GSConfig`` object to other modules to extract related configurations.
```

Comment:
Do we mention anywhere that the CLI argument will override the yaml file?

Reply:
Yes, the CLI tutorials highlight this.
https://graphstorm.readthedocs.io/en/latest/cli/model-training-inference/configuration-run.html#model-training-and-inference-configurations, and 
https://graphstorm.readthedocs.io/en/latest/cli/model-training-inference/single-machine-training-inference.html#task-specific-cli-template-for-model-training-and-inference.

Reply:
Added this information in this API doc too.